src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -51,8 +53,154 @@ and a guide for using macros can be found at
 {{ MACROS___make_filename_template(
    "raw",
    datatypes=["ecephys", "icephys"],
-   suffixes=["ecephys", "icephys", "events"]
+   suffixes=["ecephys", "icephys", "events", "channels", "electrodes","scans"]
 )
 }}
 
 <!-- Link Definitions -->
+
+{{ MACROS___make_filename_template(
+   "raw",
+   datatypes=["icephys"],
+   suffixes=["icephys", "events", "channels", "electrodes","scans","probes","coordsystem"]
+)
+}}
+
+{{ MACROS___make_filename_template(
+   "raw",
+   datatypes=["ecephys"],
+   suffixes=["ecephys", "events", "channels", "electrodes","scans","probes","coordsystem"]
+)
+}}
+
+
+#  PARTICIPANT keyfiles
+## Participant information
+
+The participants.tsv file is located at the root of the data set directory. Its presence is RECOMMENDED in order to describe information about the individual subjects (animals) from which the data was recorded. It follows the [general BIDS specifications to describe participants](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file).
+
+On top of the existing columns that can be present in this file and that are described in the BIDS specifications (participant_id, species, strain, strain_rrid, sex, handedness and age), we propose to allow adding the following ones:
+
+{{ MACROS___make_columns_table("modality_agnostic.Participants_micro") }}
+
+##  EPHYS specific files
+
+The following **metadata files are required for a ** given animal-ephys session:
```

Comment:
I think this might not render correctly with that space

```suggestion
The following **metadata files are required for a** given animal-ephys session:
```